src/storage/src/render/sinks.rs

```diff
@@ -45,13 +45,19 @@ pub(crate) fn render_sink<'g, G: Scope<Timestamp = ()>>(
     let sink_render = get_sink_render_for(&sink.connection);
 
     let mut tokens = vec![];
+
+    let snapshot_mode = if sink.with_snapshot {
+        SnapshotMode::Include
+    } else {
+        SnapshotMode::Exclude
+    };
```

Comment:
we don't yet set this correctly when restarting right? are we tracking that anywhere?

Reply:
We don't, but it's hard to do because the resume upper is not known during dataflow construction. We'll have to do some thinking on this